### PR TITLE
Test Cafe tests on the student schedule

### PR DESCRIPTION
**Pull request by doritrieur**
i have created some new tests on the student schedule, and made some functions and selectors to be generic.
please review.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "msw-client",
-  "version": "0.0.96",
+  "version": "0.0.87",
```
Comment (dimkinv) [inline package.json]: Hey Dorit.
I see there's a drop in version. 
This can create problems later in master.

File: package.json [hunk 2]
```diff
@@ -24,7 +24,9 @@
     "testcafe": "testcafe chrome tests/ -s \"tests/eyes/screenshots\"",
     "precommit": "yarn format:fix",
     "format:fix": "pretty-quick --staged",
-    "format:check": "prettier --config ./.prettierrc --list-different \"src/app/**/*{.ts, .scss}\""
+    "format:check": "prettier --config ./.prettierrc --list-different \"src/app/**/*{.ts, .scss}\"",
+    "dorit": "testcafe chrome tests/student-schedule.e2e.ts  --selector-timeout 10000 --speed 1 -r json > test.txt",
```
Comment (dimkinv) [inline package.json]: Can you please change name or remove these lines that are dedicated on your name?
Thanks

File: tests/config/config.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-
 export const testEnvironment = {
-    feUrl: 'http://127.0.0.1:4200',
-  };
+  // feUrl: 'http://127.0.0.1:4200',
```
Comment (dimkinv) [inline tests/config/config.ts]: Can you please remove comment?

File: tests/eyes/eyes.ts [hunk 4]
```diff
@@ -2,12 +2,12 @@ import { Eyes } from 'eyes.images'; // ConsoleLogHandler
 import * as readFilePromise from 'fs-readfile-promise';
 
 export class EyesDriver {
-
   eyes = new Eyes();
   constructor() {
     this.eyes.setApiKey('4ZzDF102xhdjJa3012AJPueEC4BR0I7kcoCqVGbKjFXCk110');
     //  this.eyes.setLogHandler(new ConsoleLogHandler(true));
     this.eyes.setOs('Windows 7');
+    // this.eyes.setHostOs('');
```
Comment (dimkinv) [inline tests/eyes/eyes.ts]: The tests will eventually run on Linux. Will this line somehow affect the build?

File: tests/pageobjects/class-details.po.ts [hunk 5]
```diff
@@ -7,28 +7,33 @@ export default class ClassDetailsPage {
   gradeSelectOption: Selector;
   backToClassButton: Selector;
   classRow: Selector;
-  scheduleEmptyCell: Selector;
+  // scheduleEmptyCell: Selector;
```
Comment (dimkinv) [inline tests/pageobjects/class-details.po.ts]: Here and throughout the commit. Can you please remove comments?

**Comment by dkhnn** [comment]
@doritrieur 
There are build issues on your PR.
you need to review in Travi-ci and resolve before we merge this.

**Comment by doritrieur** [comment]
@dkhnn @dimkinv i have implemented the requested changes.  pull requesting again.

**Pull request merged**